app/Http/Controllers/CategoryController.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class CategoryController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $result = [
+            'categories' => [
+                ['name' => 'sushi'],
+                ['name' => 'donburi'],
+                ['name' => 'ra-men'],
+            ],
+        ];
+        return view('category.index', compact('result'));
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        //
```

Comment:
任意ですが開発中であれば
```suggestion
        return response()->json(['message' => 'Not Implemented.], 501);
```
などで後で実装する旨を同じチームの人に伝えても良いかと思います。
https://tt-computing.com/lara6-status-code
(リリース後に残しておくと情報公開になってしまいますが)

Reply:
確かに今のままだと伝わりにくいと思うので、実装予定の旨がチームに伝わるように、ステータスコードを変更するなりして分かりやすいように変更しようと思います。

Reply:
commit: [7c4b4f7](https://github.com/xAyumux/sushi-order-system-Laravel-yumemi/pull/1/commits/7c4b4f71456014f777aabe63fd1b2804a4cc473d)
実装予定のレスポンスが設定されていない箇所に、その旨が分かるようにレスポンスを作りました。